app/views/content/train-to-be-a-teacher/teacher-training-interview.md

```diff
@@ -0,0 +1,106 @@
+
+These tips can you prepare for a successful interview. The exact process will vary depending on the teacher training provider.
+
+## Confirm your interview date
+
+The provider will email you with an interview date. If you can't make it then you may be able to ask for an alternative.
+
+Once the date is agreed you should recieve a confirmation email from the provider. Check this carefully to make sure you know exactly what to expect. For example, most providers ask for some type of presentation. Others may require maths or English tests to be completed in advance.
+
+## Prepare what you need to know
+
+It's a good idea to check the provider's website to understand more about them and what appeals to you. You can refer to this information at your interview.
+
+Consider some possible safeguarding scenarios and how you might address them. Make sure you've read the [statutory guidance for schools and colleges on safeguarding children](https://www.gov.uk/government/publications/keeping-children-safe-in-education--2).
+
+Also, take some time to find out about the [teachers’ standards](https://www.gov.uk/government/publications/teachers-standards) and the [national curriculum](https://www.gov.uk/government/collections/national-curriculum).
+
+## Get ready for common questions  
+
+The exact questions will vary from provider to provider but there are some common ones that you’re likely to be asked. It can help to prepare some clear answers to these in advance:
+
+* why do you want to teach?
+
+* what can you bring to the profession?
+
+* what are your interests and how could you bring these to the classroom?
+
+* what are some of the challenges faced by teachers today?
+
+## Consider what interviewers are looking for 
+
+Providers want to see the qualities that will help you to be a good teacher. Consider how during your interview you can show:
+
+* your passion for teaching
+
+* a professional approach
+
+* your personality when presenting
+
+* confidence, energy and enthusiasm
+
+* how you're up to the challenge of teaching
+
+* an understanding of what's involved in being a teacher
+
+## Show you’ve got the right skills
+
+Providers will also be expecting you to demonstrate you've got relevant skills, including:
+
+* how you reflect on your own performance
+
+* how you tailor information for a range of audiences
+
+* the level of your English and maths skills
+
+* the strength of your subject knowledge
+
+Many providers will ask you to do some form of presentation as they want to see you have the energy and passion to capture pupils' attention. Remember this is a chance to sell yourself and prove your commitment.
+
+## Prepare your own questions
+
+It’s good to have some questions of your own to ask. This helps you get more information and shows your interest. You could ask:
+
+* how much time will I spend in school?  
```

Comment:
the other sections talk about 'you' whereas this one talks about 'I' - do we need to tweak this to make it consistent?